README.md

```diff
@@ -56,6 +56,7 @@ raises GitHubPages::HealthCheck::Errors::InvalidCNAMEError
  :pages_domain?=>false,
  :valid?=>true
 }
+> require 'json'
```

Comment:
As currently coded, yes, JSON isn't required by default, but a method calls `to_json`, which would result in an error if not included. My thinking was, if you require `json`, `to_json` would work (just like for native objects), and if you don't you'll get an undefined method `to_json` error. Does that make sense? (use case being a simple API, that could theoretically just call `to_hash.to_json` itself).